components/timeline/index.zh-CN.md

```diff
@@ -32,6 +32,7 @@ title: Timeline
 | pending | 指定最后一个幽灵节点是否存在或内容 | boolean\|string\|ReactNode | false |
 | pendingDot | 当最后一个幽灵节点存在時，指定其时间图点 | \|string\|ReactNode | `<Icon type="loading" />` |
 | reverse | 节点排序 | boolean | false |
+| mode | By sending `alternate` the timeline will distribute the nodes to the left and right.  | `standard` \| `alternate` | `standard` |
```

Comment:
Two standard should be `left` and `right`.

Reply:
And chinese goes to 

```
通过设置 `mode` 可以改变时间轴和内容的相对位置
```

Reply:
wow sorry, bad copy & paste.